<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/res.qrc">
    <normaloff>:/icon/this</normaloff>:/icon/this</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableWidget" name="configTable">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Server</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Latency</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Local</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>&amp;Add</string>
     </property>
     <property name="icon">
      <iconset resource="../../res/res.qrc">
       <normaloff>:/icons/list-add</normaloff>:/icons/list-add</iconset>
     </property>
     <addaction name="actionManually"/>
     <addaction name="actionPaste"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRemove"/>
    <addaction name="actionShare"/>
    <addaction name="actionTestLatency"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnection"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionEdit"/>
   <addaction name="actionRemove"/>
   <addaction name="actionShare"/>
   <addaction name="actionTestLatency"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
  </widget>
  <widget class="QDockWidget" name="logDock">
   <property name="windowTitle">
    <string>&amp;Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="logArea"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/network-connect</normaloff>:/icons/network-connect</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/network-disconnect</normaloff>:/icons/network-disconnect</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/document-edit</normaloff>:/icons/document-edit</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionShare">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/document-share</normaloff>:/icons/document-share</iconset>
   </property>
   <property name="text">
    <string>&amp;Share</string>
   </property>
   <property name="toolTip">
    <string>Share</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/document-import</normaloff>:/icons/document-import</iconset>
   </property>
   <property name="text">
    <string>&amp;Import from gui-config.json</string>
   </property>
   <property name="toolTip">
    <string>Import from gui-config.json</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/document-export</normaloff>:/icons/document-export</iconset>
   </property>
   <property name="text">
    <string>&amp;Export to gui-config.json</string>
   </property>
   <property name="toolTip">
    <string>Export to gui-config.json</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/application-exit</normaloff>:/icons/application-exit</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/help-about</normaloff>:/icons/help-about</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionManually">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/edit-guides</normaloff>:/icons/edit-guides</iconset>
   </property>
   <property name="text">
    <string>&amp;Manually</string>
   </property>
   <property name="toolTip">
    <string>Add connection manually</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/list-remove</normaloff>:/icons/list-remove</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/view-refresh</normaloff>:/icons/view-refresh</iconset>
   </property>
   <property name="text">
    <string>Re&amp;fresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/edit-paste</normaloff>:/icons/edit-paste</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>&amp;Show</string>
   </property>
   <property name="toolTip">
    <string>Show window</string>
   </property>
  </action>
  <action name="actionTestLatency">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/flag-green</normaloff>:/icons/flag-green</iconset>
   </property>
   <property name="text">
    <string>Test Latency</string>
   </property>
   <property name="toolTip">
    <string>Test Latency</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
